gateway-ha/pom.xml

```diff
@@ -19,6 +19,25 @@
         <app.main.class>io.trino.gateway.ha.HaGatewayLauncher</app.main.class>
     </properties>
 
+    <dependencyManagement>
```

Comment:
Why are we using dependencyManagement?

Reply:
Dropwizard needs its own libraries versions for it's dependencies to work properly. This is the recommended way of adding them to a project.

Reply:
I think we can get rid of this here because we have the same block in the parent pom.

Reply:
agreed

Reply:
And if anything .. we would need to add the relevant dependencies specifically in the dependencies section